corehq/apps/accounting/tasks.py

```diff
@@ -192,7 +201,11 @@ def warn_subscriptions_still_active(based_on_date=None):
         is_active=True,
     )
     for subscription in subscriptions_still_active:
-        log_accounting_error("%s is still active." % subscription)
```

Comment:
It seems like it would have been much more succinct to change `log_accounting_error` rather than all the places it's called. What do you think of (mostly) reverting this PR and doing this instead?
```py
def log_accounting_error(message, show_stack_trace=False):
    if show_stack_trace:
        # must be collected before AccountingError is raised
        exc_info = sys.exc_info()
    else:
        exc_info = None
    try:
        raise AccountingError(message)
    except Exception:
        logger.error("[BILLING] %s", message, exc_info=exc_info)


class AccountingError(Exception):
    pass
```
This will also make `show_stack_trace=True` show a meaningful traceback.